config/configmodels/configmodels.go

```diff
@@ -238,4 +253,17 @@ func (proc *ProcessorSettings) IsEnabled() bool {
 	return !proc.Disabled
 }
 
+func (tlsCreds *TLSCredentials) ToGrpcServerOption() (opt grpc.ServerOption, err error) {
```

Comment:
I don't know that this function really should live in this file - anyone have suggestions on where it might better live?